src/views/v2/list.php

```diff
@@ -12,25 +12,55 @@
  * @version TBD
  *
  */
+use Tribe\Events\Views\V2\Rest_Endpoint;
 
+/**
+ * @todo  remove this and properly enqueue assets
+ */
+tribe_asset_enqueue( 'tribe-events-views-v2-manager' );
+
+$events_label_plural = tribe_get_event_label_plural();
 $events = $this->get( 'events' );
 
-$this->template( 'events-bar' );
+?>
+<form
+	class="tribe-common tribe-events tribe-events-container"
```

Comment:
reason why we're using `tribe-events-container` here?

Reply:
My goal is to have a views manager JS that will be initialized on any element that uses that class. Should we use another naming?

Reply:
I'd be partial to naming this `tribe-events-js` or something similar. However, I also don't have a good idea of the larger picture of the JS of the new views (not just views manager) and it would be tough for me to say something about it without understanding that bigger picture.

Reply:
I really don't know how I feel about these classes with `js` included. Would love to talk about the particular naming of the container better.

Reply:
Would it make sense in this upcoming JS work to utilize the `data-js` attribute approach we do on agency? It's performant and very nicely provides a clear notice to our users who may hack on things as they customize that a given element should be treated a bit more carefully as it is utilized for JS driven functionality.

Reply:
^ @paulmskim @faction23 @bordoni